unit/goto-instrument/cover_instrument.cpp

```diff
@@ -0,0 +1,20 @@
+#include <goto-instrument/cover_instrument.h>
+#include <testing-utils/use_catch.h>
+
+TEST_CASE("cover_intrument_end_of_function")
+{
+  // Arrange
```

Comment:
:pick: I'd add one for no function call (check for the invariant)

Reply:
But then it crashes the test? I'd have to start throwing exceptions to be able to test that, no? I really would want to avoid adding more of these "failed as expected" tests which pollute the output.